server/server.go

```diff
@@ -817,16 +862,32 @@ func (s *Server) configureAccounts() error {
 		for _, si := range acc.imports.services {
 			if v, ok := s.accounts.Load(si.acc.Name); ok {
 				si.acc = v.(*Account)
+				// TODO: Not sure if it is possible for an account to have a
```

Comment:
Its is possible now for latency tracking.

Reply:
Yeah.. that because I cherry-picked the commit where this fix was done, and the change to this comment was done in the PR's last commit (base on code review). Will update.